src/linkAttributes/livePreview.ts

```diff
@@ -79,19 +79,24 @@ export function buildCMViewPlugin(app: App, _settings: SuperchargedLinksSettings
                             const tokenProps = node.type.prop(tokenClassNodeProp);
                             if (tokenProps) {
                                 const props = new Set(tokenProps.split(" "));
-                                const isLink = props.has("hmd-internal-link");
-                                const isAlias = props.has("link-alias");
-                                const isPipe = props.has("link-alias-pipe");
-
-                                // The 'alias' of the md link
-                                const isMDLink = props.has('link');
-                                // The 'internal link' of the md link
-                                const isMDUrl = props.has('url');
-                                const isMDFormatting = props.has('formatting-link');
-
-                                if (isMDLink && !isMDFormatting) {
-                                    // Link: The 'alias'
-                                    // URL: The internal link
+
+                                // Square Brackets of links both internal (`[[`, `]]`) and md link (`[`, `]`)
+                                const isMDFormatting = props.has('formatting-link') || props.has('formatting-link-string');
+                                if (isMDFormatting) return;
+
+                                // Parts of internal links
+                                const isLink = props.has("hmd-internal-link"); // [[`Note` or `|` or `Alias`]]
+                                const isAlias = props.has("link-alias"); // [[Note| `Alias`]]
+                                const isPipe = props.has("link-alias-pipe"); // [[Note `|` Alias]]
+
+                                // The 'alias' of the md link (or its brackets)
+                                const isMDLink = props.has('link'); // `[` or `Alias` or `]`(URL)
+                                // The 'internal link' of the md link (or its brackets)
+                                const isMDUrl = props.has('url'); // [Alias]`(` or `URL` or `)`
```

Comment:
I added some more documentation what these flags all mean.
I also added `formatting-link-string` to `isMDFormatting` which catches single brackets of md links. This seems to fix #80.
I moved `isMDFormatting` to the top and return on those elements. They don't need styling. Previous code had some `!isMDFormatting` conditionals, but not enough. This fixes #185